cni/pkg/ebpf/app/ambient_redirect.h

```diff
@@ -17,6 +17,15 @@
 
 #define ENABLE_IPV4
 
+enum {
+    LOG_NONE = 0,
+    LOG_FATAL = 1,
+    LOG_ERROR = 2,
+    LOG_WARN = 3,
+    LOG_INFO = 4,
+    LOG_DEBUG = 5,
+};
```

Comment:
```suggestion
enum {
    LOG_NONE = 0,
    LOG_INFO = 1,
    LOG_DEBUG = 2,
};
```
Can we have fewer levels than this? I don't think most of these are strictly necessary in eBPF.

1. What does `FATAL` mean in an eBPF context? The verifier shouldn't load anything that will actually fail catastrophically. Similarly, what does `ERROR` mean? Can we simply collapse these two into `DEBUG`?

2. I would advocate that similar to Go we collapse `WARN` and `DEBUG` - typically with eBPF progs you want to turn everything on because you're debugging, or you want to turn everything off to save cycles. There's not much use for in-between states IMO.

I don't wanna bikeshed in the log level wars too much but I don't see a use for more than `OFF/INFO` and `DEBUG` in this context. If we find we need more later we can add them, I suspect we will not.

Reply:
Agree, for most cases we're using `OFF INFO DEBUG`  in practice, but I'd just like to keep the log level be consistent with istio log level, so still keep the 5 log levels here. Anyway, it's a tradeoff of consistency and practice,  what do you think of it?

Reply:
Thanks, Ben! I've limited it to `OFF INFO DEBUG`